### PR TITLE
feat: Add custom textstyle for the name property in a golden test scenario,…

### PR DESCRIPTION
**Pull request by NRijnberg**
## Description
Add custom textstyle for the name property in a golden test scenario, this is very useful when your default bodyMedium has a white text color, and the background of the golden test is white as well, this way you can make it visible by providing your own text style

## Type of Change
- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [x] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: lib/src/golden_test_scenario.dart [hunk 1]
```diff
@@ -80,16 +86,14 @@ class GoldenTestScenario extends StatelessWidget {
         children: [
           Text(
             name,
-            style: const TextStyle(fontSize: 18),
+            style: nameTextStyle ?? const TextStyle(fontSize: 18),
```
Comment (jeroen-meijer) [inline lib/src/golden_test_scenario.dart:89]:
```suggestion
            style: const TextStyle(fontSize: 18).merge(nameTextStyle),
```

File: lib/src/golden_test_scenario.dart [hunk 2]
```diff
@@ -80,16 +86,14 @@ class GoldenTestScenario extends StatelessWidget {
...
             textHeightBehavior: const TextHeightBehavior(
               applyHeightToFirstAscent: false,
             ),
           ),
           const SizedBox(height: 8),
           ConstrainedBox(
-            constraints: constraints ??
-                GoldenTestScenarioConstraints.maybeOf(context) ??
-                const BoxConstraints(),
+            constraints: constraints ?? GoldenTestScenarioConstraints.maybeOf(context) ?? const BoxConstraints(),
```
Comment (jeroen-meijer) [inline lib/src/golden_test_scenario.dart:96]: Why this change?

**Review by jeroen-meijer — CHANGES_REQUESTED** [review]
Some minor changes but LGTM otherwise

**Comment by jeroen-meijer** [comment]
@btrautmann @Kirpal I'll leave merging and release up to you. I don't see any issues with this change. 👌🏻

**Review by jeroen-meijer — APPROVED** [review]

**Comment by btrautmann** [comment]
@NRijnberg I'm working on getting this unblocked with an upgrade to 3.16.0.

**Comment by btrautmann** [comment]
@NRijnberg I just merged a PR that bumps the min version for Flutter/dart to a version compatible with your PR, so you should be able to rebase and push. Once that's done/in, I'll be able to merge and cut a new version.

**Comment by btrautmann** [comment]
I ended up cutting 0.8.0 rather than waiting for this, but feel free to get this back into a good state and we can cut a new version.

**Comment by btrautmann** [comment]
Closing in favor of https://github.com/Betterment/alchemist/pull/132.

**Pull request closed without merging**